source/edge-server/monitor.txt

```diff
@@ -0,0 +1,232 @@
+
+.. _edge-server-status:
+
+Get Edge Server Instance Status
+-------------------------------
+
+You can get details about the status of Edge Server instance in a few ways:
+
+- View the Edge Server instance details in the Atlas UI
+- View the status details in the local UI
+- Use ``edgectl`` to check the status
+- Make a request to the relevant Admin API endpoint
+
+The available details vary depending on how you access them.
+
+.. tabs::
+
+   .. tab:: Atlas UI
+      :tabid: atlas-ui
+
+      .. include:: /includes/edge/atlas-ui-get-edge-server-list-details.rst
+
+      The information available when viewing an Edge Server instance vary
+      depending on the instance state. For details, refer to 
+      :ref:`edge-server-details`.
+   
+   .. tab:: Local UI
+      :tabid: local-ui
+
+      To access the local UI, open a web browser and go to ``localhost``
+      on the ``http-port``. The default ``http-port`` is ``80``, so
+      you can access the local UI at:
+
+      ``http://localhost:80``
+
+      You can change the port by providing an alternative port as an ``Int`` 
+      with the following ``edgectl`` flag:
+
+      .. code-block:: console
+
+         edgectl config --http-port=<YOUR-PORT-INT>
+
+   .. tab:: edgectl
+      :tabid: edgectl
+
+      .. include:: /includes/edge/edgectl-status.rst
+
+   .. tab:: Admin API
+      :tabid: api
+
+      .. include:: /includes/edge/admin-api-get-edge-server-status.rst
+
+.. _edge-server-logs:
+
+Edge Server Logs
+----------------
+
+Edge Server uses the following log types:
+
+- :ref:`authentication-logs`: When an Edge Server instance attempts to 
+  authenticate with Atlas, or clients attempt to authenticate with an Edge
+  Server instance, this generates an authentication log.
+- :ref:`logs-sync`: When an Edge Server instance syncs with Atlas, or an 
+  Atlas Device SDK client syncs with Edge Server, this generates a Device Sync
+  log.
+
+Access Edge Server Logs
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can access your logs with the Admin API's
+[PLACEHOLDER - insert Admin API details here] endpoint. Or you can view logs 
+when viewing :ref:`edge-server-details` in the Atlas UI.
+
+Forward Edge Server Logs
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can configure log forwarders to store logs in a MongoDB collection or
+send them to an external service. For more information, refer to 
+:ref:`forward-logs`.
+
+.. _edge-server-metrics:
+
+Edge Server Metrics
+-------------------
+
+An Edge Server instance tracks a variety of metrics about:
+
+- The host hardware
+- Requests
+- Wire Protocol connections, and their activity
+- Sync
+
+You can access your metrics data with the Admin API's
+[PLACEHOLDER - insert Admin API details here] endpoint. Or you can view a
+subset of these metrics when viewing :ref:`edge-server-details` in the 
+Atlas UI.
+
+The exact metrics returned may be subject to change.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 15 15 25 15 25
```

Comment:
Note for reviewer: this table is annoyingly truncating the columns and does not seem to be respecting the `widths` I am setting here. I'll look into this next week.